### ISSUE TITLE
🐞 - `HintDescribe` does not read tooltip on screen reader

### ISSUE THREAD

**Issue by waterplea** [taiga-family/taiga-ui#14534]
### Reproduction url

https://taiga-ui.dev/directives/hint-describe

### Description

Currently hints are displayed after 1000ms if user does not start to type something or focuses away. Screenreaders need tooltips to be tethered with aria-describedby and be present on screen right after the focus. We might have to refactor how we treat hints show delay and display it immediately and just delay opacity/pop-in animation. That's a big deal, we gotta be carefull there, considering hint component can be replaced with a custom component.

### Taiga UI version

5

### Browser Used

Chrome

### OS Used

macOS
